faq/index.md

```diff
@@ -52,6 +52,14 @@ and capabilities and use the errors response to let users know. This error featu
 is still pending to be included in the standard since is still in
 [discussion](https://github.com/json-api/json-api/issues/7).
 
+### Where's PUT? Can I use method *X* to do *Y*? <a href="#wheres-put" id="wheres-put" class="headerlink"></a>
+
+JSON API does not currently specify the use of the `PUT` method for any purpose.
+
+Servers may complement the base specification by providing extra capabilities and
+alternative ways of requesting certain operations (e.g., resource creation via
+`PUT` in addition to `POST`).
```

Comment:
Ok, thanks for the explanation. I was in the assumption that PUT can be used for partial updates, but not the first nor the last time I was wrong :)